src/app/payment/paymentMethod/WalletButtonPaymentMethod.tsx

```diff
@@ -36,6 +36,7 @@ interface WithCheckoutWalletButtonPaymentMethodProps {
     expiryMonth?: string;
     expiryYear?: string;
     isPaymentSelected: boolean;
+    isPaymentDataRequired: boolean;
```

Comment:
We try to keep props sorted, `isPaymentDataRequired` goes between `expiryYear` and `isPaymentSelected` sorry, my mistake, I didn't mention it before. 🙏